### ISSUE TITLE
[REQUEST] New miner recipe is over the top

### ISSUE THREAD

**Issue by haleyjd** [GabrielOlvH/Industrial-Revolution#156]
**Description:**
Previously required diamond pickaxes, now requires two netherite drill heads, which together require more netherite ingots than the entirety of a set of armor and vanilla tools. If the idea of the miner is to supplement mid-game resource gathering, well, this isn't doing that at all. No desire to grind for that much more netherite as by then I'll have no use for the resources gathered by the miner anyways. Additionally, since independent drill blocks are now required for it to work anyways, why does the mining computer unit itself need to contain drill heads? This doesn't make logical sense to me.

**Reason:**
I was intrigued by the miner after watching a video of the mod being played back in November 2020, only to find the recipe has been changed to something totally inaccessible.

**Comment by GabrielOlvH** [comment]
The Mining Rig is more of a late game item. It has been buffed and can produce more ores, faster and for longer since Novemeber. The recipe does look overkill at first but you can 4x Ancient Debris with Industrial Revolution (and once you have it you basically can get infinite netherite). It really doesn't make sense to use drill heads in the Computer recipe so I'll think of something else for an update.

**Comment by haleyjd** [comment]
Fair enough. I do think it'd be cool if there were a lower-tier miner machine though, perhaps one that runs slower and only pulls some of the basic ore types. Just an idea, as it's one of the few machines with only one level.
